src/medCore/process/medRunnableProcess.cpp

```diff
@@ -52,11 +52,17 @@ dtkAbstractProcess * medRunnableProcess::getProcess()
 
 void medRunnableProcess::internalRun()
 {
-    if (d->process) {
-        if (d->process->update() == 0)
+    if (d->process)
+    {
+        int res = d->process->update();
+
+        if (res == 0)
```

Comment:
While we are modifying this part, we could take the occasion to replace 0 by DTK_SUCCESS ?